Source/Evolve/WarpXEvolve.cpp

```diff
@@ -472,13 +466,12 @@ WarpX::OneStep_nosub (Real cur_time)
             amrex::Abort(Utils::TextMsg::Err("Medium for EM is unknown"));
         }
 
-        FillBoundaryE(guard_cells.ng_FieldSolver);
+        FillBoundaryE(guard_cells.ng_FieldSolver, WarpX::sync_nodal_points);
```

Comment:
Could you also add `WarpX::sync_nodal_points` for `FillBoundaryB` above?
In this case, we can also probably remove the `NodalSync` for the Bfield below.

Reply:
Same comment as above, I thought this was breaking CI, but I removed all calls in https://github.com/ECP-WarpX/WarpX/pull/3013/commits/263acac6662f99972fe7216625472ddb43113cc8 and we can see again how CI performs.